### ISSUE TITLE
Request: Only show coin stats for those coins supported by SmartMine

### ISSUE THREAD

**Issue by wbattel4607** [aLeX1443/SmartMine#37]
This is a low priority request, but it'd be nice if the coin stats didn't display stats for coins we can't mine on SmartMine (yet).

**Comment by aLeX1443** [comment]
It's a way for users to put pressure on the developers to add them ;)

Joking aside, it is a bit misleading so we may consider removing it.

**Comment by wbattel4607** [comment]
Maybe you could add a boolean setting so the user can choose. I suppose some users may actually prefer to see it, to know the *potential* profitability.

**Comment by aLeX1443** [comment]
That's true, we should add that to settings in a future version. Also, we're going to be moving away from Boolean values in favour of check-boxes, since they're a bit more intuitive.
